src/member.ts

```diff
@@ -261,11 +224,33 @@ export class Member implements IStateful<IMemberState> {
       const state = await this.fetchState()
       if (!options.where) { options.where = {} }
       options.where.voter = state.address
-      options.where.dao = state.dao
+      options.where.dao = state.dao.id
       const sub = Vote.search(this.context, options, apolloQueryOptions) .subscribe(observer)
       return () => sub.unsubscribe()
     })
 
     return toIOperationObservable(observable)
+  }
+
+  public setState(opts: IMemberState) {
+    isAddress(opts.address)
+    
+    if(!opts.contract) throw new Error("Contract not defined in options")
+
+    this.id = Member.calculateId({ contract: opts.contract, address: opts.address})
+
+    const daoId = opts.dao && opts.dao.id.toLowerCase()
```

Comment:
I think this evaluates to a boolean... We should instead use a ternary operator `const daoId = opts.dao ? opts.dao.id.toLowerCase() : undefined`

Reply:
I think this worked but I changed it to a ternary anyways